associations/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :student
```

Comment:
- [ ] you don't want to make it possible to change the student instance variable outside of this class, therefore I advise you to make use of the attr_reader instead of attr_accessor.

Reply:
well resolved! ✔️